app/models/concerns/columns_concern.rb

```diff
@@ -44,30 +85,37 @@ def notifications_column
     def dossier_columns
       common = [dossier_id_column, notifications_column]
 
-      dates = ['created_at', 'updated_at', 'depose_at', 'en_construction_at', 'en_instruction_at', 'processed_at']
-        .map { |column| Column.new(procedure_id: id, table: 'self', column:, type: :date) }
+      states = [dossier_state_column]
 
       non_displayable_dates = ['updated_since', 'depose_since', 'en_construction_since', 'en_instruction_since', 'processed_since']
         .map { |column| Column.new(procedure_id: id, table: 'self', column:, type: :date, displayable: false) }
 
-      states = [dossier_state_column]
+      for_export_before_date = ['archived']
+        .map { |column| Column.new(procedure_id: id, table: 'self', column:, type: :text, displayable: false, filterable: false) }
+      for_export_after_date = ['motivation']
+        .map { |column| Column.new(procedure_id: id, table: 'self', column:, type: :text, displayable: false, filterable: false) }
+
+      [common, states, for_export_before_date, dossier_dates_columns, for_export_after_date, sva_svr_columns(for_export: false), non_displayable_dates].flatten.compact
+    end
 
-      [common, dates, sva_svr_columns, non_displayable_dates, states].flatten.compact
+    def dossier_dates_columns
+      ['created_at', 'updated_at', 'last_champ_updated_at', 'depose_at', 'en_construction_at', 'en_instruction_at', 'processed_at']
```

Comment:
les colonnes `updated_at` et `last_champ_updated_at` donne un truc pas clair sur le tableau de bord de la procédure. 

![Screenshot 2024-10-28 at 13-28-25 FONDS VERT - Prévention des risques d’incendies de forêt et de végétation · demarches-simplifiees fr](https://github.com/user-attachments/assets/6a607ff4-6e3a-430b-8b7a-44f1a6978648)

Je comprends pas trop la différence ... y en a un qui comprends les annontations privées ? faudrait-il faire qq chose coté wording. Par ailleurs, j'ai peur que l'ajout du terme `Dernière` prenne trop de place.

Reply:
Vu ac Marlène. Propale : La cible pour simplifier la compréhension est de parler de date {evenement}. Ceci nous oriente sur le renommage des libéllés suviants :

```yml
depose_at: Date de dépot
last_champ_updated_at: Date de dernière modification (usager)
updated_at: Date du dernier évènement
en_construction_at: Date de passage en construction
en_instruction_at: Date de passage en instruction
processed_at: Date de traitement
```

Par soucis de compatibilité "temporaire" (le temps de faire des filtres par date bornées, entre #{debut} et #{fin}), on adopte les libéllés suivant :

```yml
updated_since: Dernier évènement depuis
depose_since: Déposé depuis
en_construction_since: En construction depuis
en_instruction_since: En instruction depuis
processed_since: Traité depuis
```

Reply:
j'en pense que c'est très clair voir mm que j'ai tout compris. 
Ptit tiquage sur `Date de dernière modification (usager)` qui me parait vraiment baleze comme nom de colonne.

Et finalement, je pense que c'est vraiment maintenant qu'on peut se permettre de renommer les noms des colonnes vu que c'est un nouveau système.

Reply:
C'est le nom le plus concis que j'ai trouvé pour exprimer clairement cette donnée (et la distinguer de "Date du dernier événement")... Au départ, on partait sur "Date de dernière modification du dossier par l'usager" :/